docs/libcurl/opts/CURLSHOPT_SHARE.3

```diff
@@ -48,8 +48,9 @@ option.
 SSL session IDs will be shared across the easy handles using this shared
 object. This will reduce the time spent in the SSL handshake when reconnecting
 to the same server. Note SSL session IDs are reused within the same easy
-handle by default. Note this symbol was added in 7.10.3 but was not
-implemented until 7.23.0.
+handle by default. The size of the cache can be changed with
+\fICURLSHOPT_SSL_SESSIONID_CACHE_SIZE(3)\fP. Note this symbol was added in
+7.10.3 but was not implemented until 7.23.0.
```

Comment:
This change makes it seem like `CURLSHOPT_SSL_SESSIONID_CACHE_SIZE` was added in 7.10.3. I think we need to reword this.

Reply:
Thanks, I have swapped the sentences.